application/testing/CMakeLists.txt

```diff
@@ -846,6 +846,10 @@ f3d_test(NAME TestInteractionNonExistentRecordDirectory ARGS --interaction-test-
 f3d_test(NAME TestInteractionRecord DATA cow.vtp ARGS --interaction-test-record=${CMAKE_BINARY_DIR}/Testing/Temporary/interaction.log NO_BASELINE)
 f3d_test(NAME TestInteractionPlay DATA cow.vtp ARGS --interaction-test-play=${CMAKE_BINARY_DIR}/Testing/Temporary/interaction.log DEPENDS TestInteractionRecord NO_BASELINE)
 
+# Command Script Test
+f3d_test(NAME TestCommandScriptMissingFile DATA dragon.vtu ARGS --command-script=${F3D_SOURCE_DIR}/Testing/data/goodScript.txt NO_BASELINE)
```

Comment:
it should have  a baseline

Reply:
In this instance, what would be the baseline be? Would it just be the expected correct output?